docs/Connections/nearby-sharing.md

```diff
@@ -0,0 +1,73 @@
+---
+id: nearby-sharing
+title: Nearby Sharing
+description: Learn about Tella's encrypted, cross-platform, fully offline feature for sharing files between Tella vaults in physical proximity. Available for Android, iOS and Desktop.
+slug: /nearby-sharing
+---
+
+# Nearby Sharing
+
+Nearby Sharing allows users to securely share files between devices using Tella. 
+
+This feature is designed centering the needs of people living in internet shutdowns, where privacy, security and reliability are crucial to keep connected. It's also helpful in rural areas, or for protests or large events where mobile networks are saturated or unavailable.
+
+Nearby Sharing in Tella is:
+- Independent of Internet: Transfers work with or without an active internet connection, by establishing a direct connection between devices using local Wi-Fi.
+- Encrypted: Files move directly from one Tella vault to another, encrypted and secure.
+- Anonymous: Connection happens locally, without trace of who you shared with, when or where.
+
+Nearby Sharing will only work in physical proximity, for users connected to the same Wi-Fi network (either a local Wi-Fi or a Hotspot).
+
+
+## When to Use Nearby Sharing {#when-to-use-nearby-sharing}
+
+Nearby Sharing may be particularly well-suited for:
+- Sharing files during an internet shutdown.
+- High risk scenarios where internet is censored and cannot be trusted as a communication channel
+- During protests, to quickly back up evidence on another person's phone.
+- To increase redundancy of data, by backing up files to a personal computer or another device.
+- To migrate to a new phone.
+
+
+:::tip
+If you need help figuring out if Nearby Sharing is best for your use-case, [contact us](/contact-us)!
+:::
+
+:::tip
+If you are trying to share files to the organization you are working with, or if you are not in the same physical location as the person you want to exchange files with, [Tella's server connections](/for-organizations) could be helpful:
+* [Open Data Kit (ODK)](/odk)
+* [Uwazi](/uwazi)
+* [Tella Web](/tella-web)
+* [Google Drive](/g-drive)
+* [Nextcloud](/nextcloud)
+* [Dropbox](/dropbox)
+:::
+
+## How to use Nearby Sharing {#how-to-use}
+
+Tab the Nearby Sharing button under the **Connections** section on the Tella Homescreen to get stated.
+
+In order to use the feature, the sender and receiver need join the same Wi-Fi network. You can connect to mobile devices using Tella (Android or iOS) or connect a mobile device to a computer. There are 2 ways to connect your devices though Wi-Fi:
+1. Using a local network, such as the Wi-Fi at your home or work, or a public Wi-Fi network.
```

Comment:
> public Wi-Fi network

until it has been verified to be a good recommendation, then public wi-fi networks feels like it could be a :::tip::: shoutout: 
> "Some public Wi-Fi networks might not work due to their configuration preventing direct transfers". 

This has been my experience with trying out peer-to-peer apps on secured networks, like at universities. That said, Nearby Sharing *could work*, and it will probably work better than what I ran into in the past (local discovery over mDNS was being blocked). Just wanted to shout it out, just in case someone tries it and is disappointed!